test/unified_format.py

```diff
@@ -1704,7 +1704,9 @@ def format_logs(log_list):
                 )
             return client_to_log
 
-        with self.assertLogs("pymongo.command", level="DEBUG") as cm:
+        # Assumes that tests only verify one logging component at a time
```

Comment:
Does this assumption hold? What happens if a test specifies multiple components?

Reply:
So far this assumption holds, but if a test specifies multiple components only the first component's logs will be checked. Our current testing implementation would require a significant overhaul if a multi-component test was written, as we'd have to figure out a way to distinguish which operations belonged to which logging components.

Reply:
Good point! I didn't realize that `assertLogs` would catch all of the logging components if we just passed it the top-level `pymongo` logger. Since it does, supporting multiple components is easy, done.